difflint/lib.py

```diff
@@ -249,7 +197,25 @@ def detect_new_diff_lint_errors(log_output):
 
 
 def main():
-    if not lint_script_exists() or not all_files_configured():
+    parser = argparse.ArgumentParser(description='Linter that will examine ' +
+        'only new changes as you commit them.')
+    parser.add_argument('-c', '--check', action='store_true',
```

Comment:
After this pull request, `--check` doesn't check for configuration files anymore anyway; they are bundled with this package. It will now only check for the external executables. Those are listed as dependencies in the `debian/control` file, but if you install this package with `setup.py install` there's no way to ensure the external executables.